releasenotes/notes/remove_probability_distributions-d30bd77f0f2b9570.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    The module :mod:`qiskit.circuit.library.probability_distributions` has been removed and no longer exist as per the deprecation notice from 0.17.0
+    (released Apr 1, 2021) via #6080 <https://github.com/Qiskit/qiskit-terra/issues/6080>`__. The module was moved to qiskit-finance:
+    `qiskit_finance.circuit.library.probability_distributions`.
```

Comment:
For this, it'd be good to get intersphinx up and running, but that's beyond the scope of this PR.  I'll probably make one to do that in the future.